### PR TITLE
MG-887 - Create certs, messages, health, consumers CLI tests

### PR DESCRIPTION
**Pull request by nyagamunene**
# What type of PR is this?
This is an enhancement because it adds cli tests for certs, messages, health, consumers.

## What does this do?
It adds cli tests for certs, messages, health, consumers.

## Which issue(s) does this PR fix/relate to?
- Related Issue #887 

## Have you included tests for your changes?
Yes, I have included tests for my changes.

## Did you document any new/modified feature?
No.

### CODE REVIEW

File: cli/things_test.go [hunk 1]
```diff
@@ -28,11 +28,15 @@ var (
...
 	relation           = "administrator"
 	all                = "all"
+	get                = "get"
 )
 
 var thing = mgsdk.Thing{
-	ID:       testsutil.GenerateUUID(&testing.T{}),
-	Name:     "testthing",
+	ID:   testsutil.GenerateUUID(&testing.T{}),
+	Name: "testthing",
```
Comment (JeffMboya) [inline cli/things_test.go:35]:
```suggestion
	Name: "testThing",
```

use camelCase variable naming i.e `domainToken`. Same for all other variables formed from compound words

File: cli/certs_test.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+				"thing",
+				thing.ID,
+				invalidToken,
+			},
+			sdkErr:        errors.NewSDKErrorWithStatus(svcerr.ErrAuthorization, http.StatusUnauthorized),
+			errLogMessage: fmt.Sprintf("\nerror: %s\n\n", errors.NewSDKErrorWithStatus(svcerr.ErrAuthorization, http.StatusUnauthorized)),
+			logType:       errLog,
+		},
+		{
+			desc: "get cert successfully by id with invalid token",
```
Comment (JeffMboya) [inline cli/certs_test.go]:
```suggestion
			desc: "get cert by id with invalid token",
```

File: cli/certs_test.go [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+			args: []string{
+				get,
+				thing.ID,
+				validToken,
```
Comment (JeffMboya) [inline cli/certs_test.go]:
```suggestion
				invalidToken,
```

File: cli/health_test.go [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+)
+
+func TestHealthCmd(t *testing.T) {
+	sdkMock := new(sdkmocks.SDK)
+	cli.SetSDK(sdkMock)
+	healthCmd := cli.NewHealthCmd()
+	rootCmd := setFlags(healthCmd)
+	service := "users"
+
+	var hl mgsdk.HealthInfo
```
Comment (JeffMboya) [inline cli/health_test.go]: use decriptive variable name.`hl` sounds cryptic. Maybe something like `healthInfo`?

File: cli/message_test.go [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+				send,
+				channel.ID,
+				message,
+				thing.Credentials.Secret,
+				extraArg,
+			},
+			logType: usageLog,
+		},
+		{
+			desc: "send message with invalid secret",
```
Comment (JeffMboya) [inline cli/message_test.go]:
```suggestion
			desc: "send message with invalid thing secret",
```

**Review by JeffMboya — CHANGES_REQUESTED** [review]

File: cli/things_test.go (hunk 1 above)
Comment (nyagamunene) [reply cli/things_test.go:35]: This is not a variable name but a name of a thing.

File: cli/certs_test.go [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+	mgsdk "github.com/absmach/magistrala/pkg/sdk/go"
+	sdkmocks "github.com/absmach/magistrala/pkg/sdk/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+const (
+	revokeCommand = "revoke"
+	issueCommand  = "issue"
+)
```
Comment (dborovcanin) [inline cli/certs_test.go]: Move all command constants to the `commands_test.go` file. Keep them in separate `const` blocks and put comments explaining to which commands they refer:
```go

// Certs commands
const (
    revokeCmd = "revoke"
    issueCmd = "issue"
)
```
Do the same for the other services. Also, group common commands, something like:
```go

// CRUD and common commands
const (
    createCmd = "create"
    updateCmd = "update"
    getCmd = "get"
    enableCmd = "enable"
    ...
)
```

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — DISMISSED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**